Sources/EdgeBridge.swift

```diff
@@ -233,7 +233,7 @@ public class EdgeBridge: NSObject, Extension {
     ///
     /// - Parameter data: context data to be cleaned
     /// - Returns: dictionary where values are only of type String, Number, or Character
-    private func cleanContextData(_ data: [String: Any?]) -> [String: Any] {
+    private func cleanContextData(_ data: [String: Any]) -> [String: Any] {
```

Comment:
Should we also update `cleanContextData` to unwrap optionals in the values? or drop them? Because while non-optional values are no longer converted to `Any?`, optional values still suffer from the same issue when evaluated in the rules engine, ex:

```swift
let optionalString: String? = "stringValue"
let event = Event(name: "Test Track Event",
                  type: EventType.genericTrack,
                  source: EventSource.requestContent,
                  data: [
                    "contextdata": [
                        "stringKey": optionalString
                    ]
                  ])
```